Sources/Plasma/PubUtilLib/plAvatar/plAvatarMgr.cpp

```diff
@@ -807,7 +807,8 @@ int plAvatarMgr::FindSpawnPoint( const char *name ) const
 int plAvatarMgr::WarpPlayerToAnother(bool iMove, uint32_t remoteID)
 {
     plNetTransport &mgr = plNetClientMgr::GetInstance()->TransportMgr();
-    plNetTransportMember *mbr = mgr.GetMember(mgr.FindMember(remoteID));
+    hsSsize_t mbrIdx = mgr.FindMember(remoteID);
+    plNetTransportMember *mbr = mbrIdx >= 0 ? mgr.GetMember(mbrIdx) : nullptr;
```

Comment:
This anti-pattern is pretty common. I wonder if it would make more sense to have a `GetMemberByID()` method to return either the member or `nullptr`.